### PR TITLE
update conditions employment

### LINKED ISSUE #2769: Replace Conditions of employment
[Wireframe](https://www.figma.com/proto/XS4Ag6GWcgdq2dBlLzBkay/Admin-v3?page-id=0%3A1&node-id=949%3A5635&viewport=-1637%2C-2263%2C0.73&scaling=min-zoom&starting-point-node-id=410%3A1984&show-proto-sidebar=1)

Conditions of employment need to be updated to reflect the fields collected on the New profile. This should be made to the NEW SearchForm.

### **OLD CoE**

image

### **New CoE**

image

### PR DESCRIPTION
**Pull request by vd1992**
resolves #2769 

to test: ensure /search is operational on your end and the filters work as expected

as a thrown-in-too, I made some seeding changes which should increase the number of people in /search, with all the default filters you'd end up with like 6 candidates which didn't give much room to play around with

labels written with [this in mind](https://github.com/GCTC-NTGC/gc-digital-talent/issues/2769#issuecomment-1161784794)

### CODE REVIEW

File: api/database/factories/PoolCandidateFactory.php
Comment (petertgiles) [inline api/database/factories/PoolCandidateFactory.php:62]: This line is quite hard to read.  Could you please reformat it?

File: frontend/common/src/constants/localizedConstants.tsx
Comment (petertgiles) [inline frontend/common/src/constants/localizedConstants.tsx:503]: I find the `SearchDescription` suffix a bit confusing here.  I know when Eric created the new search form he moved the old components into a deprecated folder.  I liked that if you did that here you wouldn't have to come up with new names them.

File: frontend/common/src/constants/localizedConstants.tsx
Comment (petertgiles) [inline frontend/common/src/constants/localizedConstants.tsx:527]: Sorry, I'm a bit confused here.  The descriptions are quite different from the wireframe.  There's not even the same number!  @Jerryescandon could you help us out, please?  Also, does the order of the options matter?
image

**Review by petertgiles — CHANGES_REQUESTED** [review]
Some questions below...

File: api/database/helpers/ApiEnums.php
Comment (petertgiles) [inline api/database/helpers/ApiEnums.php:111]: Please use the constants above instead of new strings.

File: frontend/common/src/constants/localizedConstants.tsx
Comment (vd1992) [reply frontend/common/src/constants/localizedConstants.tsx:527]: as for order, I matched it to the form on the /profile

Comment (petertgiles) [reply frontend/common/src/constants/localizedConstants.tsx:527]: Would it be possible to update the design?

File: api/database/helpers/ApiEnums.php
Comment (vd1992) [reply api/database/helpers/ApiEnums.php]: [aye aye](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3143/commits/1c8c6018a4bbe347c7742a77b3883235ffa39717)

File: api/database/factories/PoolCandidateFactory.php
Comment (vd1992) [reply api/database/factories/PoolCandidateFactory.php]: [hopefully this helps](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3143/commits/1c8c6018a4bbe347c7742a77b3883235ffa39717)

File: frontend/common/src/constants/localizedConstants.tsx
Comment (Jerryescandon) [reply frontend/common/src/constants/localizedConstants.tsx:527]: Actually the ones I have on profile (Figma) are these. and they already match. 
image

Comment (Jerryescandon) [reply frontend/common/src/constants/localizedConstants.tsx:527]: Yall wanna hop on a call for this?

**Comment by petertgiles** [comment]
Had a call with Jerbo and waiting for updated designs and decision on "work weekends".  CC @gobyrne

**Comment by gobyrne** [comment]
We're on it, but waiting for confirmation with DCMO.

**Comment by vd1992** [comment]
> We're on it, but waiting for confirmation from DCMO.

Thank you!

**Comment by Jerryescandon** [comment]
We have received green light from DCMO to **REMOVE** "Work weekends".

**Comment by vd1992** [comment]
> We have received green light from DCMO to **REMOVE** "Work weekends". 👍🗑️

Thank you!
@petertgiles I think that resolves everything

**Review by petertgiles — APPROVED** [review]
I'm happy with this now.  Nice!  Please wait for Jerbo's seal of approval on the wording before merging though.

**Comment by vd1992** [comment]
> I'm happy with this now. Nice! Please wait for Jerbo's seal of approval on the wording before merging though.

@Jerryescandon

File: frontend/common/src/constants/localizedConstants.tsx
Comment (vd1992) [reply frontend/common/src/constants/localizedConstants.tsx]: okey

**Review by petertgiles — APPROVED** [review]
:100:

**Pull request merged**
